osl/namespace_linux.go

```diff
@@ -42,16 +42,16 @@ var (
 // interface. It represents a linux network namespace, and moves an interface
 // into it when called on method AddInterface or sets the gateway etc.
 type networkNamespace struct {
-	path         string
-	iFaces       []*nwIface
-	gw           net.IP
-	gwv6         net.IP
-	staticRoutes []*types.StaticRoute
-	neighbors    []*neigh
-	nextIfIndex  int
-	isDefault    bool
-	nlHandle     *netlink.Handle
-	loV6Enabled  bool
+	path          string
+	iFaces        []*nwIface
+	gw            net.IP
+	gwv6          net.IP
+	staticRoutes  []*types.StaticRoute
+	neighbors     []*neigh
+	nextIfIndexes map[string]int
```

Comment:
Again to reduce the fix footprint, I would leave the variable name as is, even if it now is a map. At the end it is one index per interface.